common/core/desktop/tests/unit/kmnkbd/debug_api.cpp

```diff
@@ -404,23 +305,52 @@ void test_store_offsets() {
     km_kbp_state_debug_item{KM_KBP_DEBUG_GROUP_EXIT, 0, {}, {u"", &gp, nullptr, {}, 6}},
     km_kbp_state_debug_item{KM_KBP_DEBUG_END, 0, {}, {u"", nullptr, nullptr, {}, 6}},
   }));
+
+  assert(action_items(test_state, {
+    {KM_KBP_IT_BACK},
+    {KM_KBP_IT_BACK},
+    {KM_KBP_IT_BACK},
+    {KM_KBP_IT_BACK},
+    {KM_KBP_IT_CHAR, {0,}, {km_kbp_usv('e')}},
+    {KM_KBP_IT_CHAR, {0,}, {km_kbp_usv('x')}},
+    {KM_KBP_IT_END}
+  }));
 }
 
 
 
 constexpr const auto help_str = "\
-debug_api [--color] <SOURCE_PATH>\n\
+debug_api [--color] <SOURCE_PATH>|--print-sizeof\n\
 \n\
-  --color      Force color output\n\
-  SOURCE_PATH  Path where debug_api.cpp is found; kmx files are\n\
-               located relative to this path.\n";
+  --color         Force color output\n\
+  --print-sizeof  Emit structure sizes for interop debug\n\
+  SOURCE_PATH     Path where debug_api.cpp is found; kmx files are\n\
+                  located relative to this path.\n";
 
 int error_args() {
   std::cerr << "debug_api: Invalid arguments." << std::endl;
   std::cout << help_str;
   return 1;
 }
 
+int print_sizeof() {
+  std::cout << std::endl;
+  std::cout << "keyboardprocessor.h:" << std::endl;
+  std::cout << "sizeof(km_kbp_context_item): " << sizeof(km_kbp_context_item) << std::endl;
+  std::cout << "sizeof(km_kbp_context_item): " << sizeof(km_kbp_context_item) << std::endl;
```

Comment:
why twice?

Reply:
```suggestion
```

Ah, because I was copying the line and modifying it.